PersonApi.Tests/V1/Controllers/BaseControllerTests.cs

```diff
@@ -1,51 +1,51 @@
+using FluentAssertions;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.Controllers;
 using Microsoft.AspNetCore.Routing;
 using NUnit.Framework;
 using PersonApi.V1.Controllers;
-using Moq;
-using System.Security.Cryptography.X509Certificates;
-using FluentAssertions;
-using PersonApi.V1.Domain;
-using PersonApi.V1.Infrastructure;
 using System.Collections.Generic;
 
 namespace PersonApi.Tests.V1.Controllers
```

Comment:
Would you please be able to explain the reason behind making these changes if it was approved in previous PR?

Reply:
Nothing sinister. I branched from master before these changes were added. I copied in this file myself (from the base api template) and this meant that I had to merge the file I added with the file that someone else had already added with the same name. As the only difference was the names of some variables and functions I resolved the conflicts by taking one file in its entirety (because it's easier to do that) and I happened to pick the one I added.